runAttendance.js

```diff
@@ -57,9 +58,9 @@ async function runAttendance(zoomResults) {
   // send for console log
   // write absence data to googly sheets
   if (recordToGoogle) writeAbsencesToGoogleSheets(attendanceObj);
-  
   printAttendance(attendanceObj);
-  emailAttendance(attendanceObj, ['magee.mooney@galvanize.com']);
+  // emailAttendance(attendanceObj, ['magee.mooney@galvanize.com']);
+  sendEmail(attendanceObj);
```

Comment:
Currently sends only the absence report. If we want to send the full report we can modify this function.